src/App.css

```diff
@@ -1,27 +1,215 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap');
+
+/* @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap'); */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
 body {
   margin: 0;
   padding: 0;
+  background: rgb(245, 246, 250);
+  font-family: 'Roboto Slab', serif;
+}
+
+h1 {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 900;
+  color: #0290ff;
+  margin-top: 0.7rem;
+}
```

Comment:
- [ ] Please make the `font-size` of this `h1` to be `1.875rem` as specified in the Zeplin design guidelines 👍